lib/main.js

```diff
@@ -13,7 +13,7 @@ function parse (src) {
   // convert Buffers before splitting into lines and processing
   src.toString().split('\n').forEach(function (line) {
     // matching "KEY' and 'VAL' in 'KEY=VAL'
-    var keyValueArr = line.match(/^\s*([\w\.\-]+)\s*=\s*(.*)?\s*$/)
+    var keyValueArr = line.replace(/( +)?#(.+)?/, '').match(/^\s*([\w\.\-]+)\s*=\s*(.*)?\s*$/)
```

Comment:
I did not use `\s` instead of the literal whitespace because the `\s` covers other "space" characters that should not be considered (very specific cases).